plugins/BEdita/API/src/Utility/JsonApi.php

```diff
@@ -132,7 +132,13 @@ protected static function extractAttributes(array $item)
         array_walk(
             $item,
             function (&$attribute) {
-                if ($attribute instanceof \JsonSerializable) {
+                if (is_array($attribute)) {
+                    foreach ($attribute as &$value) {
+                        if (is_array($value)) {
+                            unset($value['id'], $value['object_id']);
+                        }
+                    }
+                } elseif ($attribute instanceof \JsonSerializable) {
```

Comment:
I tried adding `id` and `object_id` to the `DateRange::$_hidden` array, and it seems to work exactly the same, with the benefit of keeping `JsonApi` utility class slightly more agnostic. I'd prefer that approach instead.